server.js

```diff
@@ -1,35 +1,31 @@
 // TASK 1: Setting up server
 
 
-const http = require('http');
+// const http = require('http');
 
-const server = http.createServer((req, res) => {
-  res.statusCode = 200;
-  res.setHeader('Content-Type', 'text/plain');
-  res.end('Hello World\n');
-});
 

```

Comment:
The code is using 'MongoClient' class, but there is no import statement for 'MongoClient' in the provided code. Make sure to import 'MongoClient' before using it.

Reply:
Connected to MongoDB appeared in the terminal